### PR TITLE
add data collector

### LINKED ISSUE #60: Data Collector for toggles
I wrote a data collector for the developer toolbar, so I can see all toggles and if they are active. Are you interested in a PR for this feature?

### PR DESCRIPTION
**Pull request by fadoe**
fixes #60 

Add data collector for developer toolbar. (Don't forget to clear cache after update the bundle.)

### CODE REVIEW

File: DataCollector/ToggleCollector.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+     * @param Request $request A Request instance
+     * @param Response $response A Response instance
+     * @param \Exception $exception An Exception instance
+     */
+    public function collect(Request $request, Response $response, \Exception $exception = null)
+    {
+        $toggleData = array_map(function (Toggle $toggle) {
+            return array(
+                'name' => $toggle->getName(),
+                'conditions' => array_map(function (Condition $condition) {
```
Comment (othillo) [inline DataCollector/ToggleCollector.php]: can you use the OperatorConditionSerializer to serialize the condition to an array?

File: composer.json [hunk 2]
```diff
@@ -22,6 +22,8 @@
         "php": ">=7.0",
         "qandidate/toggle": "~1.0",
         "symfony/framework-bundle": "^2.7||^3.0||^4.0",
+        "symfony/http-foundation": "^2.7||^3.0||^4.0",
```
Comment (othillo) [inline composer.json]: these are already installed by the framework-bundle: https://github.com/symfony/framework-bundle/blob/master/composer.json#L25-L26

**Comment by othillo** [comment]
LGTM!

Would you mind adding a short description about the collector to the README?

File: composer.json (hunk 2 above)
Comment (fadoe) [reply composer.json]: Yes, you are right. And they are required from the data collector. I think that all requirements should be added to composer.json, also when they are installed by other packages.

**Comment by fadoe** [comment]
Done. And now :)

**Comment by othillo** [comment]
thank you, @fadoe !

**Pull request merged**